normandy/control/static/control/js/components/RecipePreview.jsx

```diff
@@ -1,7 +1,8 @@
-import React from 'react'
-import classNames from 'classnames'
-import composeRecipeContainer from './RecipeContainer.jsx'
-import {runRecipe} from '../../../../../selfrepair/static/js/self_repair_runner.js';
+import React from 'react';
+import classNames from 'classnames';
+import composeRecipeContainer from './RecipeContainer.jsx';
+import { runRecipe } from '../../../../../selfrepair/static/js/self_repair_runner.js';
+import NormandyDriver from '../../../../../selfrepair/static/js/normandy_driver.js';
```

Comment:
If I could make all imports relative to the static folders I would. In fact, I think I'll look into that at some point.